ledger/testing/consensusRange.go

```diff
@@ -59,6 +62,25 @@ var consensusByNumber = []protocol.ConsensusVersion{
 	protocol.ConsensusFuture,
 }
 
+func versionStringFromIndex(index int) string {
+	var version string
+	if index == len(consensusByNumber)-1 {
+		version = "vFuture"
+	} else {
+		version = fmt.Sprintf("v%d", index)
+	}
+	return version
+}
+
+// rand16 samples randomness for TestConsensusRange,
+// which tests with or without LRU Cache in ledger
+func rand16(t *testing.T) uint16 {
+	var twobytes [2]byte
+	_, err := rand.Read(twobytes[:])
+	require.NoError(t, err)
+	return binary.LittleEndian.Uint16(twobytes[:])
+}
+
```

Comment:
Not that big of a deal, but I think I'd just use `rand.Int()` directly - you don't really care that it's a uint16 from what I can tell.

Reply:
right, I will just reconstruct to sample a random boolean.

Reply:
changes in [5954351](https://github.com/algorand/go-algorand/pull/5058/commits/5954351c1d37d56708629e2dd3d3d10f889fa162)